app/services/funnel/doc_auth/register_step_from_analytics_view_event.rb

```diff
@@ -4,7 +4,7 @@ class RegisterStepFromAnalyticsViewEvent
       ANALYTICS_EVENT_TO_DOC_AUTH_LOG_TOKEN = {
         Analytics::IDV_PHONE_RECORD_VISIT => :verify_phone,
         Analytics::IDV_REVIEW_VISIT => :encrypt,
-        Analytics::IDV_FINAL => :verified,
+        'IdV: final resolution' => :verified,
```

Comment:
Hmmmm this is an interesting case, it makes me rethink our position about constants... we should file a ticket to see if we can clean this up the new world of documented methods

Reply:
Yes, this seems a little off. I'll add a ticket in the backlog to revisit constants.